tests/unit/test_grid.py

```diff
@@ -130,9 +130,9 @@ def test_compute_target_areas_without_parameters(
     message = HarmonyMessage({})
 
     # values from our SMAP fixture.
-    expected_width = 5
-    expected_height = 9
-    expected_area_extent = (-161.2811, 58.2848, -160.9077, 58.9562)
+    expected_width = 6
+    expected_height = 10
+    expected_area_extent = (-161.3278, 58.2183, -160.8610, 59.0241)
```

Comment:
All of our changes to tests include increases in the col/rows of the output grids as we would expect from getting basically one more grid cell included by expanding half a width on each side.